lib/Dancer2/Manual.pod

```diff
@@ -3090,7 +3075,7 @@ Or to request a specific layout, of course:
     };
 
 Some tokens are automatically added to your template (C<perl_version>,
-C<dancer_version>, C<settings>, C<request>, C<params>, C<vars> and, if you
+C<dancer_version>, C<settings>, C<request>, C<query_parameters>, C<body_parameters>, C<route_parameters>, C<vars> and, if you
 have sessions enabled, C<session>). Check L<Dancer2::Core::Role::Template>
```

Comment:
We can expose them as objects, so you could still call methods on them. We already do it with the `request`, `response`, etc.

If you could open an issue for that, it would be great. We could also do it and merge it before we merge this document, if you want to amend it, or we could merge a different PR that just adds that extra bit of documentation.